tools/modules/module_author.rb

```diff
@@ -90,18 +80,18 @@
 
 names = {}
 
-$framework.modules.each { |name, mod|
-  x = mod.new
-  x.author.each do |r|
-    r = r.to_s
+local_modules = JSON.parse(File.open(FILENAME).read) # get cache file location from framework?
```

Comment:
I believe [`read` might be a superior approach to `open` in this instance](https://stackoverflow.com/questions/5545068/what-are-all-the-common-ways-to-read-a-file-in-ruby), as it does not mandate later calling `close` (or using a block).

```suggestion
local_modules = JSON.parse(File.read(FILENAME)) # get cache file location from framework?
```